http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -422,7 +424,279 @@ paths:
               schema:
                 $ref: "#/components/schemas/ErrorReport"
 
-## Legacy Cluster API ##
+  ## Persistent Disk paths ##
+
+  /api/google/v1/disks:
+    get:
+      summary: List all persistent disks that the caller has access to
+      description: List all persistent disks, optionally filtering on a set of labels
+      operationId: listDisks
+      tags:
+        - disks
+      parameters:
+        - in: query
+          name: _labels
+          description: >
+            Optional label key-value pairs to filter results by. Example:
+            Querying by key1=val1,key2=val2
+            returns all persistent disks that contain the key1/val1 and key2/val2 labels (possibly among other labels).
+            Note: this string format is a workaround because Swagger doesn't support free-form
+            query string parameters. The recommended way to use this endpoint is to specify the
+            labels as top-level query string parameters. For instance: GET /api/google/v1/disks?key1=val1&key2=val2.
+          required: false
+          schema:
+            type: string
+        - in: query
+          name: includeDeleted
+          description: Optional filter that includes any persistent disks with a Deleted status.
+          required: false
+          schema:
+            type: boolean
+            default: false
+      responses:
+        "200":
+          description: List of persistent disks
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/PersistentDisk"
```

Comment:
can you name this to `listPersistentDiskResponse`...I'd like to prevent us from reusing this model for other routes so that routes can evolve independent of each other and we won't accidentally update contract for one route but end up updating multiple routes.

Reply:
+1

Reply:
Makes sense. To clarify, this is only in the swagger? Or should that be a type of its own in `DiskRoutes.scala`?

Reply:
I think both..We should have dedicated response type in code as well